lib/list-swipe-save.js

```diff
@@ -59,7 +59,7 @@ SwipeSaveList.prototype.init = function(){
     actionThreshold : 120
 	});
 
-  Fliplet.Storage.get('saved_list_' + _this.listSwipe.container.parentNode.dataset.listId, {
```

Comment:
Well you still need to define the widget `{{id}}` somewhere in the DOM in order to get it in the JS files. Unfortunately that's a constraint of how the system is built.